roles/confluent.kafka_broker/tasks/main.yml

```diff
@@ -141,28 +141,18 @@
 
 - name: Create Logs Directory
   file:
-    path: "{{kafka_broker.appender_log_path}}"
+    path: "{{kafka_broker_log_dir}}"
     state: directory
     group: "{{kafka_broker_group}}"
     owner: "{{kafka_broker_user}}"
     mode: 0770
 
-- name: Create log4j Directory
-  file:
-    path: "{{kafka_broker.log4j_file | dirname}}"
-    owner: "{{kafka_broker_user}}"
-    group: "{{kafka_broker_group}}"
-    state: directory
```

Comment:
What happens if the user changes the default logging location, starts the process, then decides to change the user and restart. By not applying the user and group ownership, does this still work?

Reply:
So out of the box the permissions on the log4j file are:
```
[root@zookeeper1 kafka]# ls -lat /etc/kafka | grep "log4j.properties"
-rw-r--r-- 1 root        root          7555 Oct  1 14:47 log4j.properties
```

This means that no matter which user is running the kafka process, it has the ability to read the log4j conf and it works.

There are two things at play, the ability to change the log dir, which yes we need to apply permissions on the log dir

and making sure whatever user is running the process can access the log4j file. And the `custom-user-plaintext-rhel` scenario validates this bc all components have their users customized and the processes still have access to the log4j files and your verify tasks which validate log files get created in the location expected still pass

Reply:
lets avoid making things world readable to make customers happy, add this task back!